### ISSUE TITLE
Resolve not working this query

### ISSUE THREAD

**Issue by dev-moonduck** [dev-moonduck/vscode-ext#8]
```sql
set a='^[a-z].*[0-9]$';
select ${hiveconf:a}
```
This one should return
```sql
set a='^[a-z].*[0-9]$';
select '^[a-z].*[0-9]$'
```
But 
```sql
set a='^[a-z].*[0-9]$';
select '^[a-z].*[0-9]
```
$ with last quote is missing when it replaced

**Comment by dev-moonduck** [comment]
https://stackoverflow.com/questions/9423722/string-replace-weird-behavior-when-using-dollar-sign-as-replacement

$ is special character for replace function

**Comment by dev-moonduck** [comment]
It is resolved in #7
